src/app/containers/InlineLink/index.jsx

```diff
@@ -25,8 +25,20 @@ const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
   if (result) {
     // the path is the first item in the array
     const path = result[0];
+    const anchor = locator.split('#')[1];
```

Comment:
```suggestion
    const anchor = new Url(locator).hash?.replace('#', '');
```
Won't the existing code cause an error where the link does not contain an anchor as you'd be indexing the 2nd element from an array with only one element?

My suggestion uses a browser API that may be slightly more readable? https://developer.mozilla.org/en-US/docs/Web/API/URL/hash

Bit annoying that a `#` is included in the return from the `hash` property, hence the need for the replace.

Reply:
My worry was internet explorer 11 not having adequate support for the `URL` api but it looks like babel is pollyfilling it quite well actually. Tested and verified on IE11 (Windows 10). So I will be using your suggestion @andrewscfc. Thank you for that

Reply:
Thanks, glad the suggestion worked out :)